include/cocaine/detail/service/node/cached.hpp

```diff
@@ -64,19 +64,19 @@ cached<T>::cached(context_t& context, const std::string& collection, const std::
 
     try {
         cache = api::storage(context, "cache");
-    } catch(const api::repository_error_t& e) {
+    } catch(const std::system_error& e) {
```

Comment:
There was a specific catch for repository errors. Now we swallow any system_error, I mean maybe we should check that category is repository_error?